templates/monitoring/jobs/openshift_monitoring_federation.yaml

```
@@ -13,6 +13,7 @@
     - 'kubelet_volume_stats_used_bytes{endpoint="https-metrics",namespace=~"{{ index .Params "namespace-prefix" }}.*"}'
     - 'kubelet_volume_stats_available_bytes{endpoint="https-metrics",namespace=~"{{ index .Params "namespace-prefix" }}.*"}'
     - 'kubelet_volume_stats_capacity_bytes{endpoint="https-metrics",namespace=~"{{ index .Params "namespace-prefix" }}.*"}'
+    - 'haproxy_backend_http_responses_total{route=~"^keycloak.*", exported_namespace=~"redhat-.*sso$"}'
```

```suggestion
    - 'haproxy_backend_http_responses_total{route=~"^keycloak.*", exported_namespace=~"{{ index .Params "namespace-prefix" }}.*sso$"}'
```
Should we use the `namepsace-prefix`  template parameter here like the others ? :thinking:

---

Thank you very much for comment. I will test it and apply. Sorry for late response, as I found mail in my private mail ... (not work, although I had both mails : ). Sorry & Thanks!

---

Done - Updated and Checked for Rhoam.
Will need check all for RHMI